UE/Application/Ports/BtsPort.cpp

```diff
@@ -94,6 +95,13 @@ void BtsPort::handleMessage(BinaryMessage msg)
             }
             break;
         }
+        case common::MessageId::CallTalk:
+        {
+            logger.logInfo("BTS CallTalk from: ",from);
+            std::string text = reader.readRemainingText();
+            handler->handleTalkTextReceived(text);
```

Comment:
I would suggest using an r-value reference in `handleTalkTextReceived`, so that we do not have to create a new local variable for the sole purpose of passing it to the function.